### DISCUSSION TITLE
Add `made with bluer` section in the Readme

### DISCUSSION THREAD

**Discussion by pythops** [bluez/bluer#139] [General]
What about a new section in the Readme to list projects  based on bluer made by the community ?

**Comment by surban** [comment]
We already have [this list](https://github.com/bluez/bluer/network/dependents) which is automatically generated and kept up-to-date by GitHub. Also crates.io is providing [reverse dependencies](https://crates.io/crates/bluer/reverse_dependencies).

Maintaining such lists in our README would increase the maintenance burden for the project, which is something I wish to avoid.

Maybe create a Reddit thread at [/r/rust/](https://www.reddit.com/r/rust/) instead?

Also there is [Awesome Rust](https://github.com/rust-unofficial/awesome-rust) which aims to be a curated list for such purposes.

**Reply by pythops** [reply]
got it, thanks
